.prettierrc

```diff
@@ -12,6 +12,17 @@
         "explicitTypes": "always",
         "semi": true
       }
+    },
+    {
+      "files": "*.sol",
```

Comment:
when is this used?

Reply:
I uses it for faster formatting when developing

Reply:
OK
good so lets keep it.